src/extension.ts

```diff
@@ -117,7 +117,10 @@ export async function activate(context: vscode.ExtensionContext) {
     }
 
     const language = languages[document.languageId];
-    const t = language.parser!.parse(document.getText()); // TODO don't use getText, use Parser.Input
```

Comment:
Removed the todo since I can't find any field Input on the parser. Also if this was simple to do I assume we would already have done it.